src/components/LicensePicker/utils.js

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+import parse from 'spdx-expression-parse'
```

Comment:
@dabutvin did a bunch of work on parsing and validating SPDX expressions to better suit the needs of ClearlyDefined (e.g., accommodating NOASSERTION). We should look at if/how that is applicable here.

Reply:
Thanks for pointing this out, i will take a look at the service to see how that could be used.